ortools/xpress/environment.h

```diff
@@ -401,6 +401,8 @@ absl::Status LoadXpressDynamicLibrary(std::string &xpresspath);
 #define XPRS_MIP_UNBOUNDED 7
 #define XPRS_OBJ_MINIMIZE 1
 #define XPRS_OBJ_MAXIMIZE -1
+#define XPRS_NAMES_ROW  1
+#define XPRS_NAMES_COLUMN 2
```

Comment:
don't put this here, because it is not output by the python script. define it in xpress_interface.cpp instead, so it will not be overriden

Reply:
I would even put it next to the function calls, because it is only useful once, and the only way of understanding it anyway is to read the function's documentation